torch/csrc/jit/jit_log.cpp

```diff
@@ -161,7 +156,7 @@ std::string jit_log_prefix(
   std::stringstream prefix_ss;
   prefix_ss << "[";
```

Comment:
Slight nit, a bunch of these single character strings should actually just be chars as a microptimization. This entire string generation though can be massively simplified with libfmt though.